spec/service_models/miq_ae_service_miq_provision_spec.rb

```diff
@@ -109,16 +109,6 @@ def invoke_ae
     expect(callback_hash[:next_time_around]).to eq('do_something_better_yet')
   end
 
-  it "#set_vm_notes" do
-    method   = "$evm.root['#{@ae_result_key}'] = $evm.root['miq_provision'].set_vm_notes"
-    @ae_method.update(:data => method)
```

Comment:
What if we instead keep the test and changed it to call `set_vm_notes` on the service model and ensure that the proper backend method is called.

Reply:
Nice Iliad reference, it's normally uppercased, but I'll take it.  😉